src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/components/step_index_pattern/step_index_pattern.tsx

```diff
@@ -220,7 +222,7 @@ export class StepIndexPattern extends Component<StepIndexPatternProps, StepIndex
         getIndices({
           http,
           getIndexTags,
-          pattern: `${query}*`,
+          pattern: `${query}`,
```

Comment:
Why was this change necessary?

Reply:
perhaps this is a mistake, the project is very huge and while I debugged the logic of this component, I could leave this error after debugging.